app/services/fact_importer.rb

```diff
@@ -5,21 +5,23 @@ class FactImporter
   attr_reader :counters
 
   def self.importer_for(type)
-    importers[type.to_s]
+    Foreman::Deprecation.deprecation_warning('2.2', 'Use ImporterRegistry#get method instead of #importer_for')
```

Comment:
Just a little clarification so people know exactly what needs to be changed:
```suggestion
    Foreman::Deprecation.deprecation_warning('2.2', 'Use ImporterRegistry#get or Foreman::Plugin.importer_registry.get methods instead of FactImporter.importer_for')
```
Same applies to the other methods below as well.

Reply:
Sure